scipy/spatial/ckdtree.pyx

```diff
@@ -478,6 +498,10 @@ cdef public class cKDTree [object ckdtree, type ckdtree_type]:
         np.ndarray               boxsize_data
         np.float64_t             *raw_boxsize_data
 
+    property size:
```

Comment:
It looks like a good idea. So my last commits e7beec6 is actually buggy when the tree is unpickled. 

Could you propose a good name for count of leafnodes? Why is the count of leafnode special?